recharges/tasks.py

```diff
@@ -17,31 +94,80 @@ class Hotsocket_Login(Task):
     name = "recharges.tasks.hotsocket_login"
 
     def run(self, **kwargs):
-        """
-        Returns the token
-        """
-        l = self.get_logger(**kwargs)
-        l.info("Logging into hotsocket")
 
-        auth = {'username': settings.HOTSOCKET_API_USERNAME,
-                'password': settings.HOTSOCKET_API_PASSWORD,
-                'as_json': True}
-
-        r = requests.post("%s/login" % settings.HOTSOCKET_API_ENDPOINT,
-                          data=auth)
-        result = r.json()
+        l = self.get_logger(**kwargs)
+        login_result = request_hotsocket_login()
+        status = login_result["response"]["status"]
         # Check the result
-        if result["response"]["status"] == \
+        if status == \
```

Comment:
This line was split so that it would fit into the 79 character limit imposed by PEP8. Now that it's been shortened, why not remove the `\` and put it on a single line for easier reading?